homeassistant/components/switchbee/__init__.py

```diff
@@ -71,3 +89,75 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 async def update_listener(hass: HomeAssistant, config_entry: ConfigEntry) -> None:
     """Update listener."""
     await hass.config_entries.async_reload(config_entry.entry_id)
+
+
+async def async_migrate_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> bool:
+    """Migrate old entry."""
+    _LOGGER.debug("Migrating from version %s", config_entry.version)
+
+    if config_entry.version == 1:
+        dev_reg = dr.async_get(hass)
+        websession = async_get_clientsession(hass, verify_ssl=False)
+        old_unique_id = config_entry.unique_id
```

Comment:
Move the assert here, or cast:
```suggestion
        old_unique_id = cast(str, config_entry.unique_id)
```